ngx-tools/src/return-array-value-from-hash/return-array-value-from-hash.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Helper function to return an array of values from an hash object
+ *
+ * @param {Array} keys The array containing the key values (number|string) to retrieve from the hash
+ * @param {Object} hash The object to pull values from
+ * @return {Array} results The array of returned values
+ *
+ */
+export function returnValuesByKeys<T>(keys: Array<string | number>, hash: { [key: string]: T }): T[] {
+
+  // Convert to strings if provided keys are numbers
+  if (typeof keys[0] === 'number') {
```

Comment:
Currently, your coverage is showing that you haven't tested the `else` of this `if`. 

For this specific case I think we should add a test that passes in an array of keys as strings: `returnValuesByKeys(['1', '2'], myHash)` since we allow it, we should have a test.

---

For future reference:

In many places testing the `else` of an if doesn't really provide anything. In those cases we can tell the coverage reporter to ignore the else like this:

```typescript
// istanbul ignore else
if (true) {
  ...
}
```